### ISSUE TITLE
Unable to setup test_dataset

### ISSUE THREAD

**Issue by tanay2001** [Academic-Hammer/SciTSR#30]
The TrainDataset function used to generate test_data is broken , it accepts a rel folder as input while no such folder exists in test.
Even TrainInferDataset is never called but has been imported from loader.py , I feel this is a bug  please look into it

**Comment by brabbit61** [comment]
@tanay2001 faced the same error, just use ~1000 examples from training set to test set. It doesn't affect the final accuracy as the loss stops decreasing after ~5 epochs anyway
